session_4/blog/app/controllers/articles_controller.rb

```diff
@@ -92,6 +91,30 @@ def destroy
     end
   end
 
+  def like
```

Comment:
See if you can merge the actions for like and dislike into one action `flip_like` or something similar - Every article is "not liked" by default and each time you make a request - you flip the state